src/main/scala/group/holmes/analytics/actors/AnalyticsSupervisor.scala

```diff
@@ -0,0 +1,28 @@
+package group.holmes.analytics.actors
```

Comment:
which copyright holder are you talking about?  what is the alternative name proposal?